data/Hints/useless.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name":                  "useless",
+    "gui_name":              "Useless",
+    "description":           "Only junk hints.",
+    "add_locations":         [],
+    "remove_locations":      [],
+    "add_items":             [],
+    "remove_items":          [],
+    "dungeons_woth_limit":   2,
+    "dungeons_barren_limit": 1,
+    "named_items_required":  "False",
```

Comment:
Going by the line `if len(world.item_hints) > 0 and world.hint_dist_user['named_items_required']:` you want `false` here, not `"False"`. The latter is a non-empty string which will evaluate to `True`. The other files should likewise have `true` here to avoid confusion.

Reply:
Changed to booleans as suggested, tested on useless, tournament, and custom distributions to make sure all cases work as expected.

Test plando:
```
{
    "settings": {
      "item_hints": [
            "Progressive Strength Upgrade",
            "Progressive Strength Upgrade",
            "Lens of Truth"
        ]
    },
    "randomized_settings": {},
    "starting_items":      {},
    "item_pool":           {},
    "entrances":           {},
    "locations":           {}
}
```